config/serverless.yml

```diff
@@ -13,6 +13,8 @@ xpack.cloud.base_url: "https://cloud.elastic.co"
 xpack.cloud.profile_url: "/user/settings"
 xpack.cloud.billing_url: "/billing"
 xpack.cloud.organization_url: "/account"
+xpack.cloud.performance_url: "/performance/"
+xpack.cloud.users_and_roles_url: "/users-and-roles/"
```

Comment:
@sixstringcode I've added those 2 links but they are probably wrong.
I leave them for now until we have the good ones.

@sebgl A heads up that those 2 links will also have to be injected by cloud.

Reply:
@sebelga ack. Would it make sense for someone in your team to add these 2 lines by raising a PR in https://github.com/elastic/project-controller/blob/main/internal/application/kibana/cloud_config.go#L56 whenever the corresponding settings have been made available in Kibana and deployed in QA+staging?

Reply:
Are those urls *every* going to change? 

`baseUrl` for sure depends on the environment, so we need that injected by the controller, but can't those static url suffixes (that are all appended to the baseUrl) just live in the our serverless config file for now?

Reply:
> Would it make sense for someone in your team to add these 2 lines by raising a PR

@sebgl I can surely do that unless we decide to keep them in Kibana from Pierre's comment.

> Are those paths ever going to change?

It's a good question. Who is responsible to update those if/when cloud decides/needs to make a change to their routes? It seems to me that it should be on Cloud side. At the same time they need to be aware and remember the injection to not only update their route but also the config.

Reply:
We probably want en e2e test here and make sure none of our cloud link points to a 404 in case the route would ever change. 
cc @Dosant

Reply:
> We probably want en e2e test here and make sure none of our cloud link points to a 404 in case the route would ever change.

I'd like to check with @elastic/kibana-qa before considering adding something like this into an existing test_serverless suite. It must be not as straightforward as this.

Reply:
> can't those static url suffixes (that are all appended to the baseUrl) just live in Kibana's serverless config file for now

@sebgl I see that you thumbed up (is that a verb? 😄 ). Does that mean that you think that Kibana should own those links and we remove them from the config file you pointed above?

I just wonder how the maintenance of those links will happen over time.

Reply:
> @sebgl I see that you thumbed up (is that a verb? 😄 ). Does that mean that you think that Kibana should own those links and we remove them from the config file you pointed above? I just wonder how the maintenance of those links will happen over time.

Yeah I was thinking that having that piece of configuration live in Kibana config (removing the indirection) may actually make things simpler. In any case we cannot introduce a breaking change of those URLs just like that as the impact would be quite important. If we introduce a breaking change we'd need to coordinate it with the Kibana team in both cases I suppose.
OTOH having them in the project-controller codebase makes the breaking change more obvious/discoverable on control plane codebase side. I have mixed feelings overall 😄 I don't feel very strongly about it. It looks like at least half these URLs already live in the settings and are populated from control-plane code already, so it might be simpler to stick to the current pattern for consistency. Doing half the settings one way and half the other way is probably the worst option we have.

Reply:
Yeah makes sense! 👍 
I removed the links from the Kibana `serverless.yml` (https://github.com/elastic/kibana/pull/161971/commits/de41cd2ec3fc288ac5c008c47fa9ecade747aaa2) and created an issue (https://github.com/elastic/kibana/issues/162127) to inject the links once we have the URLs